scss/iframes/_base.scss

```diff
@@ -1,5 +1,6 @@
 [data-herman-label],
-[data-herman-th] {
+[data-herman-th],
+.color-name {
```

Comment:
I wondered about that too. Seems like a good idea, tho it will require a few more style adjustments:

![Screen Shot 2022-11-10 at 6 10 35 PM](https://user-images.githubusercontent.com/552316/201225245-9e5b9f69-36e5-4022-a506-6b9a1b96e86f.jpg)

Reply:
Makes sense to just use the `[data-herman-label]` here. fixed in https://github.com/oddbird/sassdoc-theme-herman/pull/389/commits/eabc826c15b8e1da75a529f7985f19204aaaa67f

Reply:
@dvdherron I like this visually, but I think we should probably replace the other `.color-*` selectors with `[data-herman-*]` selectors in `iframes/_color.scss` as well, right? That way the stylesheet we're loading in iframes has herman-specific selectors, and we don't risk potential conflicts with project styles. (This also fits the pattern we're using in the other iframe stylesheets, e.g. `_font.scss` and `_size.scss`.)

Reply:
Agreed. Fixed these in https://github.com/oddbird/sassdoc-theme-herman/pull/389/commits/260cbb785d2743a0da4f082ce6358e9c893b0b76